### PR TITLE
[PDMO-51] normalize header font colour and add location menu

### PR DESCRIPTION
**Pull request by Ewanjiru**
# JIRA TICKET NAME:

PDMO-51 [Modify the display of error messages on the PDMO Observations](https://issues.openmrs.org/browse/PDMO-51)

# SUMMARY:

Currently the application has different header fonts and colors in comparison with other OWA modules. These will be changed to be standard.
Also add the missing location dropdown.

### CODE REVIEW

**Review by jacksono — COMMENTED** [review]
Looks good

File: app/js/common/header.jsx [hunk 1]
```diff
@@ -8,97 +8,162 @@
...
+          });
+        }
+      );
       this.getUri();
     });
 
-    apiCall(null, 'get', '/session').then((response) => {
-      this.setState({currentUser: response.user.display});
+    apiCall(null, 'get', 'appui/session?v=full').then((response) => {
+      this.setState({
```
Comment (CeciliaWanjiku) [inline app/js/common/header.jsx]: you could destructure here and get what you need from the res as const like eg
`const {currentUser , currentLocation etc } = response`

Comment (Ewanjiru) [reply app/js/common/header.jsx]: @CeciliaWanjiku  I see what you mean.  However destructure will not apply for above as the name from response is different.

**Comment by Ewanjiru** [comment]
i

File: app/js/common/header.jsx (hunk 1 above)
Comment (Ewanjiru) [reply app/js/common/header.jsx]: @CeciliaWanjiku  on rechecking, it was possible.

Resolved.

File: app/js/common/header.jsx [hunk 2]
```diff
@@ -8,97 +8,174 @@
...
+                href="#"
+                role="button"
+                aria-haspopup="true"
+                aria-expanded="false"
+              >
+                <i className="fa fa-user" /> {' ' + this.state.currentUser}
+                <span className="caret" />
+              </a>
+              <ul className="dropdown-menu user">
+                <a className="account" href={this.state.contextPath + "/adminui/myaccount/myAccount.page"}>My Account</a>
```
Comment (chibujax) [inline app/js/common/header.jsx]: I think this line is longer than 80

**Review by chibujax — COMMENTED** [review]
Looks good as you handled well the location change.

File: app/js/common/header.jsx (hunk 2 above)
Comment (Ewanjiru) [reply app/js/common/header.jsx]: working on it. 
Thanks

**Comment by chibujax** [comment]
Forgot to squash your commits? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by Ewanjiru** [comment]
@andela-cuchendu  Please recheck.  Commits are all squashed.

**Comment by Ewanjiru** [comment]
@andela-cuchendu  feedback on the long line implemented.
Thanks.

**Comment by chibujax** [comment]
😄 You just squashed.

**Comment by chibujax** [comment]
Nice one. Looks good

**Comment by Ewanjiru** [comment]
Hehe! must have been when I was implementing your feedback

**Pull request merged**